cpp/benchmarks/io/json/nested_json.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <benchmarks/common/generate_input.hpp>
+#include <benchmarks/fixture/rmm_pool_raii.hpp>
+#include <nvbench/nvbench.cuh>
```

Comment:
Do we need this file to be `cu` file in order to include this header?

Reply:
is there a `hpp` version of this file available?

Reply:
You only need to be `.cu` file if you are doing device calls. 
Including a `.cuh` file is fine as long as you are not using any of the device functions (or data) in it.

Reply:
Should we move this include right before `#include <cstdlib>` since `nvbench` header is further than cudf ones?